test/verify/check-shell-host-switching

```diff
@@ -123,6 +129,22 @@ class TestHostSwitching(MachineCase, HostSwitcherHelpers):
         self.allow_restart_journal_messages()
         self.allow_hostkey_messages()
 
+    def testSSH(self):
```

Comment:
Each new test here is *very* expensive, as it provides three new VMs. For that reason, please merge this into `testBasic`. I think a nice place would be in line 206, before the "Test change user, not doing in edit to reuse machines" comment.